app/styles/modules/_settings.scss

```diff
@@ -604,6 +604,17 @@ section.modal-panel {
       @media screen and (max-width: 400px) {
         width: 100%;
       }
+
+      &.verified {
```

Comment:
I tried to put this selector inside the existing media query, SASS didn't like it one bit.